pyproject.toml

```diff
@@ -59,8 +59,8 @@ requests = "^2.26.0"
 pre-commit = "^2.14.0"
 pytest-benchmark = "^3.4.1"
 freezegun = "^1.1.0"
-opentelemetry-api = "^0.17b0"
-opentelemetry-sdk = "^0.17b0"
+opentelemetry-api = "^1.4.1"
+opentelemetry-sdk = "^1.4.1"
```

Comment:
I guess this might be an annoying upgrade for anyone who is using `opentelemetry==0.17b0`?

Reply:
it could be, maybe we can handle both versions? we only have two places where we had to change our code, so maybe we can do it